docs/excel/excel-add-ins-shapes.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Work with Shapes using the Excel JavaScript API
+title: Work with shapes using the Excel JavaScript API
 description: ''
-ms.date: 04/30/2019
+ms.date: 07/19/2019
```

Comment:
nit: should be 07/17/2019

Reply:
Well... it probably won't get published until then :-)